proposals/dictionary-expressions.md

```diff
@@ -249,25 +157,33 @@ HashSet<string> values = [StringComparer.CaseInsensitive, .. names];
 
 ### Question: Specialized comparer syntax.
 
-Should there be more distinctive syntax for the comparer?  Simply starting with a comparer could be difficult to tease out.  Having a syntax like so could make things clearner:
+Should there be more distinctive syntax for the comparer?  Simply starting with a comparer could be difficult to tease out.  Having a syntax like so could make things clearer:
 
 ```c#
+// `comparer: ...` to indicate the purpose of this value
 Dictionary<string, int> caseInsensitiveMap = [comparer: StringComparer.CaseInsensitive, .. existingMap];
+
+// Semicolon to more clearly delineate the comparer
+Dictionary<string, int> caseInsensitiveMap = [StringComparer.CaseInsensitive; .. existingMap];
+
+// Both?
+Dictionary<string, int> caseInsensitiveMap = [comparer : StringComparer.CaseInsensitive; .. existingMap];
 ```
 
-### Question: Other types of comparers.
+### Question: Types of comparers supported.
 
 `IEqualityComparer<T>` is not the only comparer type used in collections.  `SortedDictionary<,>` and `SortedSet<,>` both use an `IComparer<T>` instead (as they have ordering, not hashing semantics).  It seems unfortunate to leave out `SortedDictionary<,>` if we are supporting the rest.  As such, perhaps the rules should just be that the special value in the collection be typed as some `IComparer<T>` or some `IEqualityComparer<T>`.  
 
+
 ## Conversions
 
 *Collection expression conversions* are updated to include conversions to *dictionary types*.
 
-An implicit *collection expression conversion* exists from a collection expression to the following *dictionary types*:
+An implicit *collection expression conversion* exists from a *collection expression* to the following *dictionary types*:
 * A *dictionary type* with an appropriate *[create method](#create-methods)*.
 * A *struct* or *class* *dictionary type* that implements `System.Collections.IEnumerable` where:
   * The *element type* is determined from a `GetEnumerator` instance method or enumerable interface.
-  * The *type* has an *[applicable](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11642-applicable-function-member)* constructor that can be invoked with no arguments (*or* a constructor with a single parameter whose type is convertible to `IEqualityComparer<TKey>`), and the constructor is accessible at the location of the collection expression.
+  * The *type* has an *[applicable](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/expressions.md#11642-applicable-function-member)* constructor that can be invoked with no arguments (*or* a constructor with a single [*comparer*](Comparer-support) parameter), and the constructor is accessible at the location of the collection expression.
```

Comment:
>(Comparer-support) [](http://example.com/codeflow?start=249&length=18)

If these are links to the "Comparer support" section of this document, the link should be <code>(#Comparer-support)</code>.

Reply:
done.